### ISSUE TITLE
[Bug]:  After running sysbench for 10 hours continuously, the failure rate of update_pk/point_select scenarios is up to 40%

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#8788]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 7120770
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="488" alt="企业微信截图_f4877f24-391f-43b4-a5b0-c5f7d4065898" src="https://user-images.githubusercontent.com/77312370/228717239-fba9fd7f-e173-4a9f-b397-949b2cda95a8.png">
<img width="673" alt="image" src="https://user-images.githubusercontent.com/77312370/228717280-e3495b27-dfde-4f52-820a-6323b4c0dff2.png">

### Additional information

[mo-load.zip](https://github.com/matrixorigin/matrixone/files/11107316/mo-load.zip)

**Comment by sukki37** [comment]
maybe related to https://github.com/matrixorigin/matrixone/issues/8775

**Comment by aptend** [comment]
not found problem is gone in the nightly testing after fix #8789

**Comment by sukki37** [comment]
not repro for 3 days
